nuclia/sdk/kb.py

```diff
@@ -19,17 +18,29 @@ def __init__(self):
         self.search = NucliaSearch()
 
     @kb
-    def list(self, *, ndb: NucliaDBClient):
+    def list(self, **kwargs):
```

Comment:
why not having it as a regular argument? like it was before

Reply:
Because the `@kb` decorator is now implemented with `@wraps` which does check the parameters when calling the method.
So when doing `nuclia kb list`, it complains with `Missing required flags: {'ndb'}`.

The thing is the user is never meant to pass `ndb` because it is set automatically by the `@kb` decorator.